test/specs/actions/faye.spec.js

```diff
@@ -342,11 +342,76 @@ describe('Faye Actions', () => {
     });
 
     describe('subscribe', () => {
-        it('should call setFayeSubcription', () => {
-            mockedStore = createMockedStore(sandbox, generateBaseStoreProps());
-            mockedStore.dispatch(fayeActions.subscribe()).then(() => {
-                setFayeSubscriptionSpy.should.have.been.calledOnce;
+        let getClientStub;
+        let subscribeStub;
+
+        beforeEach(() => {
+            subscribeStub = sandbox.stub().resolves();
+
+            getClientStub = sandbox.stub().returnsSyncThunk({
```

Comment:
I'd like to rewrite the utils to be like `returnsThunk().val(val)`  for sync and `returnsThunk().resolves(val)` or `returnsThunk().rejects(val)` for async..... bu that's for another day...